src/android/BluetoothLePlugin.java

```diff
@@ -1056,6 +1056,10 @@ private void enableAction(CallbackContext callbackContext) {
       return;
     }
 
+    //Get Bluetooth adapter via Bluetooth Manager before enable it
+    Activity activity = cordova.getActivity();
+    BluetoothManager bluetoothManager = (BluetoothManager) activity.getSystemService(Context.BLUETOOTH_SERVICE);
+    bluetoothAdapter = bluetoothManager.getAdapter();
```

Comment:
Reading through the issue and looking at the surrounding code, I don't see why this is necessary. bluetoothAdapter will already be set when it reaches this code. Can you explain this in a bit more detail?

Reply:
Rand, you know. I've just reviewed the issue and you are totally right. Please discard the PR and sorry for the time spend. Congrats on the plugin!